test/java/io/envoyproxy/envoymobile/engine/EnvoyConfigurationTest.kt

```diff
@@ -28,7 +28,7 @@ class EnvoyConfigurationTest {
   @Test
   fun `resolving with default configuration resolves with values`() {
     val envoyConfiguration = EnvoyConfiguration(
-      false, "stats.foo.com", null, 123, 234, 345, 456, 321, "[hostname]", true, 222, 333, 567, 678, 910, "v1.2.3", "com.mydomain.myapp", "[test]",
+      false, "stats.foo.com", null, 123, 234, 345, 456, 321, "[hostname]", false, false, 222, 333, 567, 678, 910, "v1.2.3", "com.mydomain.myapp", "[test]",
```

Comment:
This test historically contains validation for all overrides.

```suggestion
      false, "stats.foo.com", null, 123, 234, 345, 456, 321, "[hostname]", true, true, 222, 333, 567, 678, 910, "v1.2.3", "com.mydomain.myapp", "[test]",
```

Reply:
`true` values are tested below.

Reply:
Right, I meant I dont think there is a need to add a new test for testing the override as long as this test does it.